data/hai/unfettered 2 war preparations.txt

```diff
@@ -0,0 +1,2164 @@
+				to display
+					has "<uhai> 20: Bigeminate Sigil: Cloak: done"
+			`	He seems to get emotional, for what must be the first time since you saw him. You can feel his pride. "This ship means a lot to me, and with it I am able to fulfill the task given to me by the Chiefs. That is mostly thanks to the cloaking ability."`
+				goto questions
+			label weapon
+			action
+				set "unfettered: weapon choice"
+			`	"I have heard tales from the Unfettered humans of a weapon that neutralizes the target's engines. Knowing it could only come from the [automated, soulless] killing machines to the east, I went looking for such weapons under the [assurance, safety] of my cloak. I expected to battle for it, but the killing machines are fighting other killing machines. I just watched them battle from a distance and then swiftly looted a ship... Except it [self destructed, exploded] in my face. So I looked for another one, and that one did not explode. I took the weapon and came home, [proud, victorious]. I had barely enough space in my cargo hold for it."`
+				to display
+					not "<uhai> 19: Detainer For Translator: done"
+			`	"Well, as you know, you brought the Detainer to me in exchange for the translation device."`
+				to display
+					has "<uhai> 19: Detainer For Translator: done"
+			label otherweapons
+			`	"For the other weapons, it was loot [taken, collected] during the Korath raids that we have held onto. The Sunbeams are the most recent addition to the ship, which were looted from ships in the north. And then of course, our anti-missile systems are cheap and effective at countering large volumes. Last, I'm sure you know plenty about flamethrowers. They are crude, but [effective, powerful]."`
+			`	"Most of us prefer not to use weapons of [aliens, untrue] because they think it means those are [more refined than, superior to] ours. I just recognize their [advantages, utility] in some scenarios."`
+				goto questions
+			label enough
+			`	"I have answered enough of your questions. Some things should remain [a mystery, concealed]. Knowledge is power, after all."`
+				to display
+					"unfettered rep: He-Yi" < 3
+			label clearconditions
+			action
+				clear "unfettered: questions"
+				clear "unfettered: sword choice"
+				clear "unfettered: hull choice"
+				clear "unfettered: cloak choice"
+				clear "unfettered: weapon choice"
+			`	"The time will come soon, prepare yourself for [war, tragedy]: bring us Jump Drives, train in challenge battle, and when we are ready and only then, we will strike."`
+
+event "joined the unfettered"
+	government "Escort"
+		swizzle 4
+	"reputation: Wanderer" <?= -1
+	set "global: unfettered human"
```

Comment:
Global? Does this mean that any new pilot that is started is also an unfettered human?

Reply:
nah I just add this for a future achievement